js/app/filters/importerrorfilter.js

```diff
@@ -29,17 +29,29 @@ app.filter('importErrorFilter', function () {
 		}
 
 		//TODO - use n instead of t to use proper plurals in all translations
-		switch(file.errors) {
-			case 0:
-				return t('calendar', 'Successfully imported');
-
-			case 1:
+		if (file.errors === 0) {
+			return t('calendar', 'Successfully imported');
+		} else if(file.errors === 1) {
+			if (file.duplicates === 1) {
+				return t('calendar', 'Partially imported, skipped 1 duplicate');
```

Comment:
1 -> one

Reply:
I don't think we can have both, `n` and `one`